webviz_subsurface/plugins/_line_plotter_fmu/controllers/build_figure.py

```diff
@@ -143,6 +148,93 @@ def _update_plot(
     def _clear_real_highlight(_click: int) -> Optional[List]:
         return []
 
+    @app.callback(
+        Output(get_uuid("stored_x_value"), "data"),
+        Output(get_uuid("traces"), "options"),
+        Output(get_uuid("traces"), "value"),
+        Output(get_uuid("statistics_warning"), "children"),
+        Input(
+            {
+                "id": get_uuid("data_selectors"),
+                "data_attribute": "x",
+                "source": "table",
+            },
+            "value",
+        ),
+        State(
+            {
+                "id": get_uuid("data_selectors"),
+                "data_attribute": "ensemble",
+                "source": "table",
+            },
+            "value",
+        ),
+        State({"id": get_uuid("parameter-filter"), "type": "data-store"}, "data"),
+        State(get_uuid("traces"), "value"),
+    )
+    def _update_statistics_options(
+        x_column_name: str,
+        ensemble_names: List,
+        parameter_filter: Dict,
```

Comment:
```suggestion
        parameter_filter: Optional[Dict],
```
according to line 184?